util/max_stock_heap.go

```diff
@@ -0,0 +1,9 @@
+package util
```

Comment:
if you're going to use these heaps we need to move them to a different package, util is getting overbloated